src/docfx/lib/git/GitUtility.cs

```diff
@@ -47,6 +47,20 @@ public static bool IsGitRepository(string path)
             return false;
         }
 
+        public static bool IsDefaultBranch(string branch)
```

Comment:
Consider a different name since this is not technically checking for default branch, e.g., the default branch for docfx is `dev`.

Reply:
😂

---
In reply to: [478139519](https://github.com/dotnet/docfx/pull/6453#discussion_r478139519) [](ancestors = 478139519)

Reply:
but in docfx, for all our consumed repository, we actually use the `master`/`main` as the default banch. so would this be a concern?

---
In reply to: [478140410](https://github.com/dotnet/docfx/pull/6453#discussion_r478140410) [](ancestors = 478140410,478139519)